modules/javafx.controls/src/main/java/javafx/scene/control/ScrollPane.java

```diff
@@ -73,8 +73,12 @@
  * node in a Group.
  * <p>
  * ScrollPane sets focusTraversable to false.
- * </p>
- *
+ * <p>
+ * ScrollPane by default only acts on key presses when it has the focus
+ * ({@link #isFocused()} returns {@code true}) and won't respond to unconsumed
+ * key events that bubble up from a focused child control. The key presses it
+ * acts on are platform specific, but by default should include keys for panning
```

Comment:
which keys are platform specific?  I could not find any - the behavior registers no platform-specific bindings.

Reply:
Perhaps it should read "may be platform specific" -- I just don't want to pin FX down on a specific implementation, as FX is aligning with the platforms it runs on, which it doesn't control.  So if one of the platforms at some point starts using different keys, then I'd expect FX to align with that.

Reply:
right.  it just feels to me like the sentence "The key presses..." is out of place as it does not specify something that SP has to do.  remove?

Reply:
I've removed the entire sentence, and also the "by default" part.